lib/urlwatch/filters.py

```diff
@@ -252,6 +252,53 @@ def filter(self, data, subfilter=None):
         element_by_id.feed(data)
         return element_by_id.get_html()
 
+class ElementByclass(html.parser.HTMLParser):
```

Comment:
This is mostly a copy of `ElementByID` -- please refactor it so that the class is called e.g. `ElementByAttribute` and pass in `"id"` or `"class"` as constructor parameter in the `GetElementById` and `GetElementByClass` filters.